easybuild/easyconfigs/a/amdsmi/amdsmi-25.4.2-GCCcore-14.2.0-ROCm-6.4.1.eb

```diff
@@ -0,0 +1,69 @@
+easyblock = 'CMakeMake'
+
+name = 'amdsmi'
+_rocm_version = '6.4.1'
+version = '25.4.2'
+versionsuffix = f'-ROCm-{_rocm_version}'
+
+homepage = 'https://github.com/ROCm/amdsmi'
+description = """
+The AMD System Management Interface (AMD SMI) library offers a unified tool for managing and monitoring
+GPUs, particularly in high-performance computing environments. It provides a user-space interface that
+allows applications to control GPU operations, monitor performance, and retrieve information about the
+system's drivers and GPUs."""
+docurls = ['https://rocm.docs.amd.com/projects/amdsmi/en/latest/']
+
+toolchain = {'name': 'GCCcore', 'version': '14.2.0'}
+
+source_urls = [
+    'https://github.com/ROCm/amdsmi/archive/refs/tags/',
+    'https://github.com/amd/esmi_ib_library/archive/refs/tags/'
+]
+sources = [
+    f'rocm-{_rocm_version}.tar.gz',
+    'esmi_pkg_ver-4.1.2.tar.gz',
```

Comment:
I think it may be good to add a comment about determining the right version of this package (looks like you need to find it in amdsmi's CMakeLists.txt?), and maybe even add a variable for the version of this package?